pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -225,15 +226,20 @@ private boolean hasParticipantConsentedToSharedLearning(ElasticSearchParticipant
                     .orElse(false);
         }
         int age = DateTimeUtil.calculateAgeInYears(dateOfBirth);
-        if (age >= 7) {
+        boolean hasCompletedPediatricConsentAddendum = participant.hasCompletedActivity(CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_CODE);
+
+        if (hasCompletedPediatricConsentAddendum) {
+            if (age >= 7) {
```

Comment:
I'm surprised we have not defined this age as a constant. Does it only apply to this situation?

Reply:
This was a little unclear/surprising to me as well.  There is a ticket to rework some of this (both DSS and DSM) [here](https://broadworkbench.atlassian.net/browse/PEPPER-1264).